website/content/docs/sync/azurekv.mdx

```diff
@@ -5,3 +5,89 @@ description: The Azure Key Vault destination syncs secrets from Vault to Azure.
 ---
 
 # Azure Key Vault
+
+The Azure Key Vault destination enables Vault to sync and unsync secrets of your choosing into
+an external Azure account. When configured, Vault will actively maintain the state of each externally-synced
+secret in realtime. This includes sending new secrets, updating existing secret values, and removing
+secrets when they either get dissociated from the destination or deleted from Vault.
+
+## Setup
+
+A service principal with a client id and client secret will be needed to configure Azure Key Vault as a
+sync destination. This [guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
+will you through creating the service principal.
+
+Once the service principal is created, the next step is to
+[grant the service principal](https://learn.microsoft.com/en-us/azure/key-vault/general/rbac-guide?tabs=azure-cli)
+access to Azure Key Vault. We recommend using the "Key Vault Secrets Officer" built-in role,
+which gives sufficient access to manage secrets.
+
+## Usage
+
+When Azure credentials for the service principal have been obtained and it has access to Azure Key Vault,
+Vault can be configured to create an Azure destination, then begin to associate your secrets with this destination.
+
+1. Store any KV secret into Vault, like so:
+  ```shell-session
+  $ vault kv put secret/foo mypass=bar
+  ```
+
+  Output:
+  ```
+  === Secret Path ===
+  secret/data/foo
+
+  ======= Metadata =======
+  Key                Value
+  ---                -----
+  created_time       2023-09-21T20:52:47.723315Z
+  custom_metadata    <nil>
+  deletion_time      n/a
+  destroyed          false
+  version            1
+  ```
+
+1.	Configure the sync destination:
+  ```shell-session
+  $ vault write sys/sync/stores/azure-kv/my-azure-1 \
+  key_vault_uri=$KEY_VAULT_URI \
+  client_id=$CLIENT_ID \
+  client_secret=$CLIENT_SECRET \
```

Comment:
Where would be the best place to document the alternative way to pass the authentication parameters like using env vars such as `AZURE_CLIENT_SECRET`?